presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -191,6 +188,194 @@ This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
 
+
+.. _tuning-pref-hive:
+
+Tuning
+-------
+
+The following configuration properties may have an impact on connector performance:
+
+``hive.assume-canonical-partition-keys``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``false``
+ * **Description:** Disable optimized metastore partition fetching for non-string partition keys. Setting this property allows to avoid ignoring data with non-canonical partition values.
+
+
+``hive.domain-compaction-threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Integer`` (at least ``1``)
+ * **Default value:** ``100``
+ * **Description:** Maximum number of ranges allowed in a tuple domain without compacting it. Higher value will cause more data fragmentation but allows to use row skipping feature when reading ORC data. Setting this value higher may have large impact on ``IN`` and ``OR`` clauses performance in scenarios making use of row skipping.
+
+
+.. _force-local-scheduling:
+
+``hive.force-local-scheduling``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``false``
+ * **Description:** Force splits to be scheduled on the same node (ignoring normal node selection procedures) as the Hadoop DataNode process serving the split data. This is useful for installations where Presto is collocated with every DataNode and may increase queries time significantly. The drawback may be that if some data are accessed more often, the utilization of some nodes may be low even if the whole system is heavy loaded. See also :ref:`node-scheduler.network-topology<node-scheduler-network-topology>` if less strict constrain is preferred - especially if some nodes are overloaded and other are not fully utilized.
```

Comment:
sure, some grammar changes, then
s/data are/data is         -- the rest of the documentation uses "data is" rather than "data are"
s/heavy loaded/heavily loaded
s/if less strict constrain/if a less strict constraint
s/and other are/and others are